### ISSUE TITLE
satfunc2df only works for one SATNUM

### ISSUE THREAD

**Issue by eivindsm** [equinor/ecl2df#11]
Function satfunc2df fails to parse tables with more than 1 satnum due to upstream issue in sunbeam. Testfunction conveniently passes due to the fact that it only has one SATNUM in test.

**Comment by berland** [comment]
Is this a known or unknown sunbeam issue?

Should we resort to an existing Python-parser in the meantime?

**Comment by berland** [comment]
Test that exposes bug: https://github.com/equinor/ecl2df/tree/satfuncbug

**Comment by berland** [comment]
Trying to parse the swofstr without alll the band-aid in 
https://github.com/equinor/ecl2df/blob/cf1662bdafadc7944fe7d01b58f92a90b241fb98/ecl2df/eclfiles.py#L17
we get the error from sunbeam:
```
ValueError: PARSE_MISSING_DIMS_KEYWORD: Expected the kewyord: TABDIMS to infer the number of records in: SWOF
```
 Perhaps opm-common/sunbeam is unable to parse SWOF without knowing in advance the number of SATNUMs it contains?

**Comment by berland** [comment]
Confirmed by trial and error that sunbeam requires TABDIMS with NTSFUN set correctly to parse SWOF.

TABDIMS with correct NTSFUN can be injected into the string if it is not found in the string/deck. There should be an option for the user to supply it when it is known.

It might be possible for satfunc2df, when TABDIMS is not found in the deck, to estimate it from
trial and error, progressing from 1 SATNUM and upwards (and not including (PARSE_RANDOM_SLASH, sunbeam.action.ignore))

**Comment by berland** [comment]
Solved in #14
